gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/models/observability/langfuse/langfuse_setting.py

```diff
@@ -42,6 +42,11 @@ class LangfuseObservabilitySetting(BaseObservabilitySetting):
     url: AnyUrl = Field(
         description='The Langfuse server url', examples=['https://cloud.langfuse.com'], default='http://localhost:3000'
     )
+    public_url: Optional[AnyUrl] = Field(
```

Comment:
Why do you have a public URL here as it's not used in the orchestrator.

Reply:
I see we share the same data model between the orchestrator and the bot admin BotObservabilityConfigurationDTO.setting is LangfuseObservabilitySetting which came from the Orchestrator API contract we should have a BotAdmin API contract decorrelated from the ochestrator contract to avoid this.